contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol

```diff
@@ -191,8 +191,8 @@ contract DefaultReserveInterestRateStrategy is IReserveInterestRateStrategy {
   }
 
   /// @inheritdoc IReserveInterestRateStrategy
-  function calculateInterestRates(DataTypes.CalculateInterestRatesParams calldata params)
-    external
+  function calculateInterestRates(DataTypes.CalculateInterestRatesParams memory params)
+    public
```

Comment:
@miguelmtzinf is it needed?

Reply:
These changes are already part of the code (github is not calculating the diff correctly). They have been introduced per BGD's request (for the V2-V3 migration). 
Here PRs: https://github.com/aave/aave-v3-core/pull/721 and https://github.com/aave/aave-v3-core/pull/712